homeworks/magurmarian/hw01/somefunc.py

```diff
@@ -0,0 +1,6 @@
+def add(num1, num2):
```

Comment:
- This file was supposed to have the "**calculation**.py" name.
- doc-strings are missing